src/materials/MeshRefractionMaterial.tsx

```diff
@@ -6,6 +6,10 @@ import { shaderMaterial } from '../core/shaderMaterial'
 import { MeshBVHUniformStruct, shaderStructs, shaderIntersectFunction } from 'three-mesh-bvh'
 import { version } from '../helpers/constants'
 
+declare module 'three-mesh-bvh' {
+  const shaderIntersectFunction: string
+}
+
```

Comment:
This variable [is exported](https://github.com/gkjohnson/three-mesh-bvh/blob/master/src/index.js#L18-L21), but is not in `three-mesh-bvh`'s declarations.